### PR TITLE
Add a Docker image

### PR DESCRIPTION
**Pull request by filiptronicek**
- [x] Add Dockerfile
- [x] Add docs
- [x] Make the mounting of volumes work
- [x] Add the GitHub workflow step
- [x] Push to Dockerhub (for now under [filiptronicek/vscode](https://hub.docker.com/repository/docker/filiptronicek/vscode))

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+ARG NODE_VERSION=14.17.5
```
Comment (akosyakov) [inline Dockerfile:1]: that actually should not be matter, since the server is self contained

Comment (filiptronicek) [reply Dockerfile:1]: Oh right then, should we just leave it at latest?

Comment (akosyakov) [reply Dockerfile:1]: something like  ubuntu:18.04 will be more appropriate for real life usage

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -135,3 +135,29 @@ jobs:
...
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+      - name: Download Linux artifact
+        uses: actions/download-artifact@v2
+        with:
+          path: outputs/
+          name: Linux-x64
+      - name: Set server version
+        run: echo "VERSION=$(ls outputs/ | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p')" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (filiptronicek) [reply .github/workflows/release.yml:151]: Yes indeed.

**Pull request merged**